Sources/Filters/Core/PolyDataNormals/index.js

```diff
@@ -105,6 +153,13 @@ function vtkPolyDataNormals(publicAPI, model) {
 
     output.getPointData().setNormals(outputNormals);
 
+    if (model.computeCellNormals) {
+      const outputCellNormalsData =
+        publicAPI.vtkPolyDataCellNormalsExecute(input);
```

Comment:
it's too bad you are recomputing the normals that were computed when doing points normals.

Can you please modify `vtkPolyDataNormalsExecute` to optionally return cells normals ?